src/stores/store.ts

```diff
@@ -269,15 +269,25 @@ export const useStore = defineStore('store', {
         this.setNetworkMessage('dashboard.verifyingNet');
 
         await client.startFarming(this.plotPath, this.plotSizeGB);
-
         // TODO: consider moving logging to client.ts
-        util.infoLogger('PLOTTING PROGRESS | farmer started');
+        util.infoLogger('Farmer started');
+      } catch (error) {
+        // TODO: consider moving logging to client.ts
+        util.errorLogger('Farmer start error!');
+        this.setError({
+          title: 'errorPage.startFarmerFailed',
+          // TODO: replace default error message with specific one
+          message: 'errorPage.defaultErrorMessage',
+        });
+        return
+      }
 
-        const syncState = await client.getSyncState();
-        this.setSyncState(syncState);
-        let isSyncing = await client.isSyncing();
+      const syncState = await client.getSyncState();
+      this.setSyncState(syncState);
+      let isSyncing = await client.isSyncing();
 
-        do {
+      do {
+        try {
           await new Promise((resolve) => setTimeout(resolve, 3000));
           const syncState = await client.getSyncState();
```

Comment:
this is where the error happens -> `getSyncState()`. 

The reason is, farmer/node is going to be restarted, and at the moment not available. Hence, the `getSyncState()` is failing because the WebSocket is closed. 

My solution was to wrap these with `try`, and simply print&forget the error. 

What can be better is: try for like 20-30 seconds if it is creating an error, and if within 20-30 seconds, it can continue without errors, that's fine. If the error is still continuing after 20-30 seconds, then show a related error page, and ask for a restart.

My current solution does not handle this 20-30 seconds of `trial window`. It only logs the error and keeps trying again. Not the best solution, but way better than what we had before